docker-compose.yml

```diff
@@ -0,0 +1,8 @@
+version: '3'
+
+services:
+  redis:
+    image: redis
```

Comment:
It's a better idea to tag the version - at least major - of the image. If you run this today you'll get `redis:latest` which is 6.0.5 If you then run it two months from now and 6.1 is out docker-compose won't pull it because `latest` is already cached - new users will get 6.1 though.

Locking this to at least 6.0 would be a good start then bump the image to 6.1, etc which would force users who currently run it today to make sure they're staying up with everyone else

Reply:
@leoferreiralima can you add the version?

Reply:
yes, i will add as possible

Reply:
@diego3g version added